instagram/src/components/PostContainer/PostFooter.js

```diff
@@ -0,0 +1,59 @@
+import React from 'react';
+import heartIcon from '../../heart_icon.svg';
+import redHeartIcon from '../../heart_icon_red.svg';
+import speechBubbleIcon from '../../speech_bubble_icon.svg';
+import styled from 'styled-components';
+
+const PostFooterIconsWrapper = styled.div `
+    width: 100%;
+    padding: 10px 1.5%;
+    display: flex;
+    justify-content: flex-start;
+    align-items: center;
+`;
+const NumberOfLikes = styled.p `
+    padding-left: 2%;
+    font-weight: bold;
+    margin-right: 1%;
+`;
+const FooterIcon = styled.img `
+    opacity: 2;
+    margin-right: 2%;
+    cursor: pointer;
+    height: 35px;
+    ${props => (props.type === 'heart' ? `height: 29px` : null)}
+    ${props => (props.type === 'bubble' ? `height: 25px; transform: scaleX(-1)` : null)}
+`;
+
+class PostFooter extends React.Component {
+    constructor(props) {
+        super();
+        this.state = {
+          likes: props.likes,
```

Comment:
great use of passing this as props, a lot of people have trouble with this concept as it's not always easy to see if you can do it or not.